Sources/BraintreeVenmo/BTConfiguration+Venmo.swift

```diff
@@ -26,4 +26,9 @@ extension BTConfiguration {
     var venmoEnvironment: String? {
         json?["payWithVenmo"]["environment"].asString()
     }
+
+    /// Indicates whether ECD is enabled for the Venmo merchant.
```

Comment:
I think we should opt to use the full name vs acronym in our documentation - that way people don't need to hunt down what `ECD` means